packages/core/core/src/core-api/routes/validation/utils.ts

```diff
@@ -95,7 +115,16 @@ export const safeSchemaCreation = (id: Internal.UID.Schema, callback: () => z.Zo
     // Replace the placeholder with the real schema
     safeGlobalRegistrySet(id, schema);
 
-    strapi.log.debug(`Schema ${transformedId} successfully created and registered`);
+    // Show completion for user content only
+    if (!isBuiltInSchema) {
+      const fieldCount = Object.keys((schema as any)?._def?.shape || {}).length || 0;
+      const schemaName = transformedId
+        .replace('Document', '')
+        .replace('Entry', '')
+        .replace(/([A-Z])/g, ' $1')
+        .trim();
+      strapi.log.info(`   ✅ "${schemaName}" schema created with ${fieldCount} fields`);
```

Comment:
I would keep all of these new `info` logs as `debug` too, possibly even lower (`silly`?) since it's weird to see these on every startup. If it only happened once it would make sense, but since it happens every time it's just kind of annoying 😆